go/vt/vtctl/reparentutil/util_test.go

```diff
@@ -1243,7 +1243,7 @@ func TestFindPositionForTablet(t *testing.T) {
 				}{
 					"zone1-0000000100": {
 						Position: &replicationdatapb.Status{
-							Position:              "unused",
```

Comment:
This PR means we can't have an invalid position here. I can ignore the `error` this returns, but I don't think this is realistic